pkg/org/user_provider.go

```diff
@@ -50,12 +49,6 @@ func (p *JWTProvider) Auth(req bunrouter.Request) (*User, error) {
 
 	user, err := SelectUserByEmail(ctx, p.app, email)
 	if err != nil {
-		if err == sql.ErrNoRows {
-			user := &User{
-				Email: email,
-			}
-			user.Init()
-		}
```

Comment:
AFAIR the idea here is that users should be able to sign their own JWT tokens for Uptrace, i.e. to authenticate users in some proxy and then redirect them to Uptrace. This might work as cheap alternative to SAML if you willing to writing such proxy.